src/reducers/Setting.js

```diff
@@ -1,13 +1,20 @@
 import {
+  DEFAULT_LANGUAGE,
   HEADER_BACKGROUND_COLOR,
 } from '../types';
 
 const INITIAL_STATE = {
+  defaultLanguage: 'en',
```

Comment:
This should not be `defaultLanguage`, but `language`. The default language should be a string in a config file that is then set to language. Conceptually, it does not make sense.